src/main/kotlin/kotliquery/Session.kt

```diff
@@ -29,7 +30,7 @@ open class Session(
 
     private fun PreparedStatement.setParam(idx: Int, v: Any?) {
         if (v == null) {
-            this.setObject(idx, null)
+            this.setNull(idx, Types.VARCHAR)
```

Comment:
If I understand correctly, this call doesn't work for non-string columns.

Reply:
As a matter of fact, there's no other non-intrusive way to figure out the supposed type of current `null` value. And Postgres only relies on this information when the type of param cannot be figured out differently.

Reply:
Maybe just my case-matching didn't work out.